src/cpp/core/include/core/r_util/RActiveSessionStorage.hpp

```diff
@@ -51,25 +53,33 @@ namespace r_util {
       const std::string propertiesDirName_ = "properites";
       const std::string fileSessionDirPrefix_ = "session-";
 
+      FilePath buildPropertyDir(const std::string& id);
       FilePath buildPropertyPath(const std::string& id, const std::string& name);
```

Comment:
Nit: In order to contrast more with `buildPropertyDir` I might change `buildPropertyPath` to `buildPropertyFile` although I might change them both to `get` instead of `build`. Also, is it possible to make them `const`?